### PR TITLE
UI updates for Search Bar, Info text and Fees

### PR DESCRIPTION
**Pull request by dimak1**
*Issue #:* /bcgov/entity#14607

*Description of changes:*
- UI updates for Search Bar, Info text and Fees
- Refactor rows/cols structure for Search Bar
- Update tests

Screen Shot 2023-01-05 at 5 14 50 PM

Screen Shot 2023-01-05 at 5 14 41 PM

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/src/components/search/SearchBar.vue [hunk 1]
```diff
@@ -14,149 +14,175 @@
...
-              persistent-hint
-              placeholder="Last Name"
-              v-model="searchValueLast"
-              @keypress.enter="searchCheck()"
+        </v-row>
+        <div v-if="typeOfSearch">
+          <div class="pt-0 pb-1">
+            <div v-html="typeOfSearch" class="font-weight-bold search-title"></div>
+          </div>
+        </div>
```
Comment (cameron-eyds) [inline ppr-ui/src/components/search/SearchBar.vue]: Not so sure about these layers of divs, are they really all required? Why not 1 div? or 2 at most perhaps?

**Review by cameron-eyds — APPROVED** [review]
LGTM!
Just a minor comment

**Review by doug-lovett — APPROVED** [review]
Looks good.

File: ppr-ui/src/components/search/SearchBar.vue (hunk 1 above)
Comment (dimak1) [reply ppr-ui/src/components/search/SearchBar.vue]: Right, there were rows/cols before. Will simplify.

**Pull request merged**